### DISCUSSION TITLE
第2节课作业(6班)

### DISCUSSION THREAD

**Discussion by maxchiron** [InternLM/Tutorial#273] [作业]
第2节课作业(6班)

**Comment by JackieGLQ** [comment]
微信名：老高
基础作业：https://z01krqqa831.feishu.cn/docx/EMs2dSD0YoQpSExNKDHc3dY0n9g#FmiEdEVmHoRCREx3VLAcA6XjnEb
进阶作业：https://z01krqqa831.feishu.cn/docx/EMs2dSD0YoQpSExNKDHc3dY0n9g#StrKd5A3doCXmixuYotcX5ronTf

**Comment by zhangzefeng92** [comment]
微信名称：捂上眼睛看世界

基础作业：

    使用 InternLM-Chat-7B 模型生成 300 字的小故事（需截图）。
![294763358-1b9c861a-47d7-4f0f-8d77-78ac4b4e6e78](https://github.com/InternLM/tutorial/assets/58167242/4ae8fb3d-55b6-4d54-a498-6d4c3445ad59)

    熟悉 hugging face 下载功能，使用 huggingface_hub python 包，下载 InternLM-20B 的 config.json 文件到本地（需截图下载过程）。
![294767215-5c8725db-d900-4481-a83c-f1272d5c044d](https://github.com/InternLM/tutorial/assets/58167242/d2be7e14-78fe-4721-94c6-768f70d19165)

注：huggingface国内无法访问解决方案https://hf-mirror.com/

进阶作业

    完成浦语·灵笔的图文理解及创作部署（需截图）
    ![294769264-8b31ba6d-e1d8-469f-b584-0e595df18cfb](https://github.com/InternLM/tutorial/assets/58167242/75e9496d-b7b7-4536-b05e-57621c111822)

注：Could not create share link. Missing file: /root/.local/lib/python3.10/site-packages/gradio/frpc_linux_amd64_v0.2. 解决方案

        Download this file: https://cdn-media.huggingface.co/frpc-gradio-0.2/frpc_linux_amd64
        Rename the downloaded file to: frpc_linux_amd64_v0.2
        Move the file to this location: /root/.local/lib/python3.10/site-packages/gradio
        chmod 777 /root/.local/lib/python3.10/site-packages/gradio/frpc_linux_amd64_v0.2

    完成 Lagent 工具调用 Demo 创作部署（需截图）
    ![Uploading 294764708-14b1de7a-ad18-431b-8da8-4a530bf4fd91.png…]()

**Comment by yokebtc** [comment]
yoke - [https://app.yinxiang.com/fx/fa88c3db-ec90-4389-a7db-a78f26038ac3](https://app.yinxiang.com/fx/fa88c3db-ec90-4389-a7db-a78f26038ac3)

**Comment by 99312254** [comment]
![写故事](https://github.com/InternLM/tutorial/assets/37920955/2dbb0641-3c09-428e-9c4f-57c6110015da)
![image](https://github.com/InternLM/tutorial/assets/37920955/794ef26e-faaf-4846-b9b9-215fc8a48b9e)
![image](https://github.com/InternLM/tutorial/assets/37920955/34cd6ff1-a16f-4b96-bb31-00e8e93184a8)
![image](https://github.com/InternLM/tutorial/assets/37920955/14ad0305-828f-4d55-b48f-bdaa97e57674)
"https://blog.csdn.net/cq99312254/article/details/135625924
https://blog.csdn.net/cq99312254/article/details/135654220
https://blog.csdn.net/cq99312254/article/details/135656348"
